app/styles/ui/check-runs/_ci-check-run-list-item.scss

```
@@ -30,4 +30,16 @@
     width: 250px;
     height: 12px;
   }
+
+  .view-on-github {
+    display: none;
+    padding: var(--spacing);
+    padding-bottom: var(--spacing-half);
+    color: var(--text-secondary-color);
+  }
+
+  &:hover .view-on-github,
+  .view-on-github.show {
+    display: block;
+  }
```

It feels weird that when you hover over the icon/button to view it on GitHub it doesn't highlight too 🤔  This feels better to me:

```suggestion
  &:hover .view-on-github,
  .view-on-github.show {
    display: block;
  }
  
  .view-on-github:hover {
    color: var(--text-primary-color);
  }
```